service-impl/redirector-service-impl/rest-redirector-service/src/test/java/uk/gov/gchq/palisade/redirect/service/impl/RESTRedirectorV1IT.java

```diff
@@ -212,7 +203,7 @@ public void shouldSetHostNameCorrectly() throws Exception {
             url.connect();
 
             //Then
-            assertThat(redirector.host, either(is(equalTo(InetAddress.getLocalHost().getCanonicalHostName()))).or(is(equalTo("localhost"))));
+            assertThat(redirector.host, either(is(equalTo(InetAddress.getLocalHost().getCanonicalHostName()))).or(is(equalTo("localhost"))).or(is(equalTo("hostname"))).or(is(equalTo("127.0.0.1"))));
```

Comment:
This shouldn't be "hostname" as a literal keyword, the test should be for InetAddress.getLocalHost().getHostName()